{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2018-1060", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "python before versions 2.7.15, 3.4.9, 3.5.6rc1, 3.6.5rc1 and 3.7.0 is vulnerable to catastrophic backtracking in pop3lib's apop() method.  An attacker could use this flaw to cause denial of service.", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 14:43:14 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2019-08-21", "cve_detail": "", "references": [{"URL": "https://bugs.python.org/issue32981", "Description": "CONFIRM:https://bugs.python.org/issue32981"}, {"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-1060", "Description": "CONFIRM:https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-1060"}, {"URL": "https://docs.python.org/3.5/whatsnew/changelog.html#python-3-5-6-release-candidate-1", "Description": "CONFIRM:https://docs.python.org/3.5/whatsnew/changelog.html#python-3-5-6-release-candidate-1"}, {"URL": "https://docs.python.org/3.6/whatsnew/changelog.html#python-3-6-5-release-candidate-1", "Description": "CONFIRM:https://docs.python.org/3.6/whatsnew/changelog.html#python-3-6-5-release-candidate-1"}, {"URL": "https://support.hpe.com/hpsc/doc/public/display?docLocale=en_US&docId=emr_na-hpesbst03951en_us", "Description": "CONFIRM:https://support.hpe.com/hpsc/doc/public/display?docLocale=en_US&docId=emr_na-hpesbst03951en_us"}, {"URL": "https://www.debian.org/security/2018/dsa-4306", "Description": "DEBIAN:DSA-4306"}, {"URL": "https://www.debian.org/security/2018/dsa-4307", "Description": "DEBIAN:DSA-4307"}, {"URL": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/46PVWY5LFP4BRPG3BVQ5QEEFYBVEXHCK/", "Description": "FEDORA:FEDORA-2019-51f1e08207"}, {"URL": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JSKPGPZQNTAULHW4UH63KGOOUIDE4RRB/", "Description": "FEDORA:FEDORA-2019-6e1938a3c5"}, {"URL": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/AEZ5IQT7OF7Q2NCGIVABOWYGKO7YU3NJ/", "Description": "FEDORA:FEDORA-2019-cf725dd20b"}, {"URL": "https://lists.debian.org/debian-lts-announce/2018/09/msg00030.html", "Description": "MLIST:[debian-lts-announce] 20180925 [SECURITY] [DLA 1519-1] python2.7 security update"}, {"URL": "https://lists.debian.org/debian-lts-announce/2018/09/msg00031.html", "Description": "MLIST:[debian-lts-announce] 20180926 [SECURITY] [DLA 1520-1] python3.4 security update"}, {"URL": "https://access.redhat.com/errata/RHBA-2019:0327", "Description": "REDHAT:RHBA-2019:0327"}, {"URL": "https://access.redhat.com/errata/RHSA-2018:3041", "Description": "REDHAT:RHSA-2018:3041"}, {"URL": "https://access.redhat.com/errata/RHSA-2018:3505", "Description": "REDHAT:RHSA-2018:3505"}, {"URL": "https://access.redhat.com/errata/RHSA-2019:1260", "Description": "REDHAT:RHSA-2019:1260"}, {"URL": "http://www.securitytracker.com/id/1042001", "Description": "SECTRACK:1042001"}, {"URL": "https://usn.ubuntu.com/3817-1/", "Description": "UBUNTU:USN-3817-1"}, {"URL": "https://usn.ubuntu.com/3817-2/", "Description": "UBUNTU:USN-3817-2"}], "cve_repaire": "", "editor": "warcup", "add_time": "2018-06-18"}